corehq/motech/dhis2/templates/dhis2/dataset_map_list.html

```diff
@@ -29,6 +29,7 @@ <h2>{% trans "DHIS2 DataSet Maps" %}</h2>
   <div class="col-xs-12 col-sm-8 col-md-8 col-lg-6">
     <div id="send-now-result"
          class="text-success hide"></div>
+    <a id="remote-logs" data-bind="attr: {href: hrefUrl}">View log</a>
```

Comment:
this needs to be translated
I don't see a reference to the `hrefUrl` property anywhere. Am I just missing it?

Reply:
Strange...I removed this. Maybe I reverted at one point and forgot to change this again.